src/rootPages/Designer/ui_work_object_workspace_popupNewDataField.js

```diff
@@ -72,108 +81,156 @@ export default function (AB) {
             id: ids.component,
             resize: true,
             modal: true,
-            height: 500,
-            width: 700,
+            boarderless: true,
+            height: 503,
+            width: 692,
             head: {
                view: "toolbar",
                css: "webix_dark",
-               cols: [
+               paddingX: 2,
+               elements: [
                   {
                      view: "label",
-                     label: L("Add new field"),
-                     css: "modal_title",
+                     id: ids.title,
                      align: "center",
+                     label: L("<span>Choose Field-Type</span>"),
+                     css: "modal_title"
                   },
                   {
-                     view: "button",
-                     autowidth: true,
-                     type: "icon",
-                     icon: "nomargin fa fa-times",
-                     click: () => {
-                        this.buttonCancel();
-                     },
-                     on: {
-                        onAfterRender() {
-                           UIClass.CYPRESS_REF(this);
-                        },
-                     },
-                  },
+                     cols: [
+                       {
+                           view: "button",
+                           id: ids.buttonMaximize,
+                           label: L("<span class=\"webix_icon\"><i class=\"nomargin fa fa-expand\"></i></span>"),
+                           css: "webix_transparent",
+                           width: 40,
+                           click: () => {
+                              this.buttonMaximize();
+                           }
+                       },
+                       {
+                           view: "button",
+                           id: ids.buttonMinimize,
+                           label: L("<span class=\"webix_icon\"><i class=\"nomargin fa fa-compress\"></i></span>"),
```

Comment:
I think if we are not going to translate this to another language you do not have to wrap in the L() function.